tests/e2e/yacat.spec.ts

```diff
@@ -17,7 +17,7 @@ describe("Password cracking", function () {
     const mask = "?a?a";
     const hash = "$P$5ZDzPE45CigTC6EY4cXbyJSLj/pGee0";
     executor = await TaskExecutor.create({
-      package: "golem/examples-hashcat:legacy",
+      package: "055911c811e56da4d75ffc928361a78ed13077933ffa8320fb1ec2db",
```

Comment:
Probably worth leaving a comment here for future generations why this test must use a specific hash rather than the latest image